packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -119,15 +147,16 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     styles,
     layout,
     onRenderAvatar,
-    showMuteIndicator
+    showMuteIndicator,
+    dominantSpeakers
   } = props;
   const [sortedRemoteParticipants, setSortedRemoteParticipants] = useState<VideoGalleryRemoteParticipant[]>([]);
 
   const ids = useIdentifiers();
 
   useEffect(() => {
```

Comment:
Is there a reason this is inside a `useEffect`? using useEffect with `set` will mean two separate renders and looks like this is just to avoid recreating the sort every render.
Could this instead use a useRef/useMemo/useCallback (not sure which is most applicable) and be something like:
```
const sortedRemoteParticipants = useMemo(() => sortParticipants(remoteParticipants), [remoteParticipants]);
```

Reply:
Good point. I overlooked the fact this this is inside useEffect. Let me see if I can memoize it.